src/app/tests/suites/certification/Test_TC_OO_2_3.yaml

```diff
@@ -441,8 +445,8 @@ tests:
       PICS: OO.S.A4002 && OO.S.C40.Rsp
       response:
           constraints:
-              minValue: 255
-              maxValue: 345
+              minValue: 300 - PIXIT.OO.MaxCommunicationTurnaround
```

Comment:
In 11a the `OffWaitTime` countdown starts.
In first 11b `readAttribute` of `OnOff` is performed that can take up to `PIXIT.OO.MaxCommunicationTurnaround`.
In second 11b `readAttribute` of `OnTime` is performed that can take up to `PIXIT.OO.MaxCommunicationTurnaround`.
In third 11b `readAttribute` of `OffWaitTime` is performed that can take up to `PIXIT.OO.MaxCommunicationTurnaround`.
So in total I think it should be `300 - 3 * PIXIT.OO.MaxCommunicationTurnaround`.

Reply:
Updated